src/test/java/com/SAS/acceptanceTests/TeamManagementAT.java

```diff
@@ -372,9 +373,76 @@ public void addAdditionalTeamOwnerFail() {
 
     }
 
-    //Todo - Chen
     @Test
-    public void addTeamManager() {
+    public void addTeamManagerSuccess() {
+        //preparations- create a player
+        User player = userController.createUser("ItayC", "ItAY1234", "Itay Cohen", UserType.PLAYER, true, null);
+        ((Player)player).setFieldRole(FieldRole.MIDFIELDER);
+        ((Player)player).setTeam(team);
+        ((Player)player).setDateOfBirth(LocalDate.parse("1990-12-01"));
+        team.addPlayerToTeam((Player)player);
+
+        Team myTeam = ((TeamOwner) teamOwner).getTeam();
+        myTeam.getPersonalPage().showPersonalPage();
+
+        //enter editing mode
+        if (teamManagement.enterEditingMode(teamOwner, myTeam)) {
+            System.out.println("--Select to add a team manager--");
+            System.out.println("Please select a nominee from the followings:");
```

Comment:
Missing check - if the user can add team owners to the team.

Reply:
Why? It is checked inside the function.. Why check twice?

Reply:
Ok.